### ISSUE TITLE
Velocity "scroll" doesn't work

### ISSUE THREAD

**Issue by beseffort** [julianshapiro/velocity#844]
I installed "velocity-animate": "^2.0.1"
And I use it like below within Vue component.
```
import Velocity from 'velocity-animate';
...
var d = document.getElementsByClassName("chat-wrapper");
Velocity(d[0], "scroll", { duration: 500 });
...
```

But I get error in console.
`Uncaught (in promise) Error: Velocity: First argument (scroll) was not a property map, a known action, or a registered redirect. Aborting.`

I couldn't find solution. Please help me!
(FYI, I confirmed the dom element - d[0] exists.)

**Comment by Rycochet** [comment]
Scroll is now a property, but I'm not 100% happy with the internal code for it, so it's not listed in the wiki (current docs) yet. I'm going to point you at my StackOverflow answer and copy the example from it - https://stackoverflow.com/a/48847153/3328079

```javascript
$("#container").velocity({
   scrollTop: "500px"
}, {
   duration: 500
});
```


If you want to scroll to a specific element then you'll need to find its offset - there will be a decent example of this once I get the time to do it.

**Comment by beseffort** [comment]
@Rycochet , it didn't work even though there was no error.

**Comment by rootion** [comment]
Can you add some documentation on how to use scroll in V2? 
I have some client prototypes in codepen that were using velocity that are all broken now.
There is no error, as pointed above, it just doesnt work.

How should I rewrite this line?
`Velocity( target, 'scroll', [ 0.25, 0.46, 0.45, 0.94 ] );`

**Comment by Jonathan0wh** [comment]
The documentation was still not updated... I used v2.0.2 and followed the instruction in the document before finding myself in bugs...

**Comment by Oluwatemilorun** [comment]
Are you using it with jQuery? In the documentation, it was used with jquery.velocity.js
So, i think the 'scroll' is a jQuery property

**Comment by Oluwatemilorun** [comment]
@Rycochet 
`$("#container").velocity({
   scrollTop: "500px"
}, {
   duration: 500
});`
will throw a _maximum call stack size_ error (v2.0.2)

**Comment by Rycochet** [comment]
@Oluwatemilorun jQuery has got nothing to do with Velocity, and it didn't in the majority of V1 either - scroll has changed a lot since then.

Regarding that scroll bug - it should be fixed now! #866

**Comment by joelucky39** [comment]
I'm having to use jQuery because Velocity scroll does not work in vue.js with Laravel
